ballerina/Module.md

```diff
@@ -13,19 +13,19 @@ To use the Snowflake connector, you must have a valid Snowflake account. If you
 
 1. Log in to your Snowflake account.
 2. Go to the **Warehouses** tab under the **Admin** section, as shown below.
-   <img alt="Snowflake Warehouse" src="https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-snowflake/main/docs/setup/resources/snowflakes_create_warehouse.png" style="border:1px solid #000000; width:100%"/>
+   <img alt="Snowflake Warehouse" src="https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-snowflake/main/docs/setup/resources/snowflakes_create_warehouse.png"/>
```

Comment:
Shall we use the markdown syntax here since we don't need HTML tags anymore?

```suggestion
   ![Snowflake Warehouse](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-snowflake/main/docs/setup/resources/snowflakes_create_warehouse.png)
```